modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserSetDigestTest.java

```diff
@@ -85,7 +82,7 @@ private String _generateRandomEmailAddress() {
 			RandomTestUtil.randomString(), ".com");
```

Comment:
Have you checked that this will not result in a flaky test because of validation on email addresses?